Kickstarter-iOS/Views/Cells/BackerDashboardProjectCell.swift

```diff
@@ -50,6 +50,11 @@ internal final class BackerDashboardProjectCell: UITableViewCell, ValueCell {
       |> UITableViewCell.lens.isAccessibilityElement .~ true
       |> UITableViewCell.lens.accessibilityHint %~ { _ in Strings.Opens_project() }
       |> UITableViewCell.lens.accessibilityTraits .~ UIAccessibilityTraitButton
+      |> UITableViewCell.lens.contentView.layoutMargins %~~ { _, cell in
+        cell.traitCollection.isRegularRegular
+          ? .init(topBottom: Styles.grid(2), leftRight: Styles.grid(20))
+          : .init(topBottom: Styles.gridHalf(3), leftRight: Styles.grid(2))
+    }
```

Comment:
Can you update the spacing to be `.init(topBottom: Styles.grid(1), leftRight: Styles.grid(2))` for iPhone size? I realize the spacing between cells is not consistent for a lot of pages, but I think it's based on the size of the cards and this was the spacing in the original design.